Final Project/Portfolio/js/main.js

```diff
@@ -0,0 +1,7 @@
+function copyrightDate(){
```

Comment:
This is probably solid, but if you ever want to do anything else with dates check out: http://momentjs.com/